src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState } from 'react';
+import { FilterType } from '../../types/FilterType';
+import classNames from 'classnames';
+import { deleteTodo } from '../../api/todos';
+import { FooterType } from '../../types/FooterType';
+
+export const Footer: React.FC<FooterType> = ({
+  todos,
+  setTodos,
+  setCurrentFilter,
+  setError,
+  setIsLoadingIds,
+}) => {
+  const [activeFilter, setActiveFilter] = useState<string>(FilterType.All);
```

Comment:
why do you need this state here? looks redundant, just pass to the footer the currentFilter